jobapi/payloads.go

```diff
@@ -1,11 +1,13 @@
 package jobapi
 
-import "sort"
+import (
+	"sort"
 
-// Error response is the response body for any errors that occur
-type ErrorResponse struct {
-	Error string `json:"error"`
-}
+	"github.com/buildkite/agent/v3/internal/socket"
+)
+
+// ErrorResponse is the response body for any errors that occur.
+type ErrorResponse = socket.ErrorResponse
```

Comment:
would this cause problems for anyone importing the client library if it relies on an internal module?

Reply:
It shouldn't cause problems. I tested it to be sure 🤓